docs/best-practices.md

```diff
@@ -29,6 +29,18 @@ You should *NOT* use Argo Rollouts for preview/ephemeral environments. For that
 
 The recommended way to use Argo Rollouts is for brief deployments that take 15-20 minutes or maximum 1-2 hours. If you want to run new versions for days or weeks before deciding to promote, then Argo Rollouts is probably not the best solution for you.
 
+Keeping parallel releases for long times, complicates the deployment process a lot and opens several questions where different people have different views on how Argo Rollouts should work.
+
+For example let's say that you are testing for a week version 1.3 as stable and 1.4 as preview.
+Then somebody deploys 1.5
+
+* Some people believe that the new state should be 1.3 stable and 1.5 as preview
+* Some people believe that the new state should be 1.4 stable and 1.5 as preview
+
+And then let's say that 1.5 has an issue. Some people believe that Argo rollouts should "rollback" to 1.3 while other people think it should rollback to 1.4
+
+All these problems are not present if you make the assumption that each release stays active only for a minimal time and you always create one new version when the previous one has finished.
+
```

Comment:
Should we define how Argo Rollouts behaves?

Reply:
I added explanation about what Argo Rollouts does right now.